pennylane/devices/tests/test_gates.py

```diff
@@ -62,6 +62,7 @@
     "S": qml.S(wires=[0]),
     "SWAP": qml.SWAP(wires=[0, 1]),
     "T": qml.T(wires=[0]),
+    "SX": qml.SX(wires=[0]),
```

Comment:
It would be good to add some tests to `pennylane/tests/ops/test_qubit_ops.py`, e.g., `test_sx_decomposition`, which makes sure we get the decomposition right.

Reply:
added a `test_sx_decomposition`.  I think the first time around I had the order of operations reversed.